GSLUX-695: Draw toolbar

tkohr opened the pull request:
### JIRA issue

https://jira.camptocamp.com/browse/GSLUX-695

### Description

This PR adds the buttons for the draw toolbar for desktop and mobile. UI only, no functionality yet.

mki-c2c reviewed (approved):
LGTM, the checkbox 'follow routes' is visible by default, whereas on prod it only appears when the line tool is selected.
But this is probably normal since ther are no logics associated yet.

tkohr commented:
> LGTM, the checkbox 'follow routes' is visible by default, whereas on prod it only appears when the line tool is selected. But this is probably normal since ther are no logics associated yet.

Thanks for the review @mki-c2c ! Good point, I've added a commit to handle the 'follow roads' checkbox and rebased the PR.
